mitodl/open-discussions
# Add read-only podcasts API

#### Pre-Flight checklist

  - [x] Migrations are backwards compatible

#### What are the relevant tickets?
Fixes #2745 

#### What's this PR do?
Adds a serializer and read-only REST API for listing podcast data

#### How should this be manually tested?
Create some podcast data in your local database. Set `FEATURE_PODCAST_APIS=True`. Then go to `/api/v0/podcasts/` and verify that the output makes sense.

#### Any background context you want to provide?
I'm not sure what fields to include or not include so please let me know your opinions there.
